mappings/net/minecraft/entity/projectile/ArrowEntity.mapping

```diff
@@ -1,7 +1,10 @@
 CLASS net/minecraft/unmapped/C_otvmxtwh net/minecraft/entity/projectile/ArrowEntity
+	FIELD f_brxzfoon NO_EFFECT_COLOR I
 	FIELD f_guuemiic potion Lnet/minecraft/unmapped/C_almkolgw;
+	FIELD f_iaefprml EXPOSED_POTION_DECAY_DURATION I
 	FIELD f_moofqymr effects Ljava/util/Set;
 	FIELD f_vmgnarau COLOR Lnet/minecraft/unmapped/C_rinmcaxy;
+	FIELD f_xcoztqqe EVENT_POTION_UFF B
```

Comment:
Does UFF stand for something?

Reply:
It's a typo, it's supposed to be puff. I don't like it but wasn't sure what else to put

Reply:
```suggestion
	FIELD f_xcoztqqe EFFECT_PARTICLE_SPAWN_INTERVAL B
```